comeet/Data/Parsers/FreebusyBlockParser.swift

```diff
@@ -22,8 +22,9 @@ class FreebusyBlockParser {
             let endString = freebusyBlockDict[Constants.endKey] as? String else {
                 return nil
         }
-        guard let start = Date.fromAPIFreeBusy(stringDate: startString),
-            let end = Date.fromAPIFreeBusy(stringDate: endString),
+
+        guard let start = Date.fromISO8601(string: startString),
```

Comment:
Did you test this? This is the bit where I'm not 100% sure that the *output* dates from the RESTApi are ISO8601.  We can fix it up on the service side.